packages/pages/src/common/src/template/internal/getTemplateFilepaths.ts

```diff
@@ -91,7 +91,6 @@ export const getGlobalClientServerRenderTemplates = (
 
 /**
  * @param templatesRootPath the path where the templates live, typically src/templates
- * @param scopedTemplatePath the subfolder path inside templatesRoot to scope to - used in multibrand setups
```

Comment:
This param was not implemented -- not sure if it should be removed from the doc or implemented

Reply:
I think it used to exist but was removed. Good to remove the docs.